### PR TITLE
csv-files for tweet_entities

### PR DESCRIPTION
**Pull request by SabiRoth**

### CODE REVIEW

**Review by TheresaStrohmeier — APPROVED** [review]
👍

File: analysis/tweet_entities_analysis/get_tweet_entities.py [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+POLITICIANS_LIST = '../../assets/all_politicians.json'
+
+def get_politicians_list():
+    politicians = []
+    with open(POLITICIANS_LIST, 'r', encoding='utf-8') as infile:
+        for p in json.load(infile):
+            politicians.append(p)
+    return politicians
+
+politicians_list = get_politicians_list()
```
Comment (JonasPuchinger) [inline analysis/tweet_entities_analysis/get_tweet_entities.py]: Could just be

    with open(POLITICIANS_LIST, 'r', encoding='utf-8') as infile:
          politicians_list = [p for p in json.load(infile)]

File: analysis/tweet_entities_analysis/get_tweet_entities.py [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+        return url
+
+def get_detail_info(tweet):
+    info = dict.fromkeys(['name', 'screen_name','id', 'party'])
+    for p in politicians_list:
+        if p['screen_name'] == filename[:-5]:
+            info['name'] = p['Name']
+            info['screen_name'] = p['screen_name']
+            info['id'] = p['id']
+            info['party'] = p['Partei']
```
Comment (JonasPuchinger) [inline analysis/tweet_entities_analysis/get_tweet_entities.py:40]: else ???

**Review by JonasPuchinger — APPROVED** [review]
Looks good, just some small suggestions

**Pull request merged**